runtime/doc/syntax.txt

```diff
@@ -1896,6 +1896,22 @@ set "lite_minlines" to the value you desire.  Example: >
 
 	:let lite_minlines = 200
 
+LOG							*ft-log-syntax*
+
+Vim comes with a simplistic generic log syntax highlighter.  Because the log
+file format is so generic, highlighting is not enabled by default, since it
+can be distracting.  If you want to enable this, simply set the "log" filetype
```

Comment:
(off topic) I think we should enable it by default, if it doesn't slow down huge log files. But it probably does. We need a general solution for that...

Reply:
No, see vim/vim#18218